packages/ui/src/domainModules/locality/higherOrderComponents/createGetCuratedLocalityById.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
Could these be generalise to HOCs that can be used for any resource type? It looks like it to me, by passing selectors and action creators into the HOC factory. If so, we can do it next time it is needed